packages/NodeTypeResolver/DependencyInjection/PHPStanExtensionsConfigResolver.php

```diff
@@ -41,8 +41,14 @@ public function resolve(): array
         $generatedConfigDirectory = dirname($generatedConfigClassFileName);
 
         $extensionConfigFiles = [];
-
-        foreach (GeneratedConfig::EXTENSIONS as $extension) {
+        // Some extensions are unsupported and only work when used by PHPstan directly, so we skip them.
+        $unsupportedExtensions = [
+            'mglaman/phpstan-drupal', // https://github.com/rectorphp/rector/issues/7795
+        ];
```

Comment:
@mfrieling this skipping is just hiding issue, that can cause another extension still have the issue, see https://github.com/rectorphp/rector/issues/7795#issuecomment-1439648675

Better to fix it instead. Please create simple repoducible repository for it? Better with github action running so it can be resolved once for all.

Reply:
I agree with you. As I wrote already I see that just a hotfix to be able to use it as a patch and able to use rector in my project until a release with a correct fix is available. I don't expect this to be merged into the production version.

But creating the correct fix must be done by the rector dev team and not by me.

Reply:
we can't provide a fix without reproduced bug, that's why you need to spent time to create reproducible repository or unit test or e2e test so we can check.

You may read this article :) https://tomasvotruba.com/blog/2021/02/01/effective-debug-tricks-narrow-scoping/

Reply:
Understand. But as I have to do it in my free time (it's not budgeted in the customer project so I don't get paid for it), it can take 2 or 3 months before I find the time, if I don't forget it by then :-(